src/sw.ts

```diff
@@ -456,16 +429,14 @@ async function fetchHandler ({ path, request, event }: FetchHandlerArg): Promise
   event.request.signal.addEventListener('abort', abortFn)
 
   try {
-    const { id, protocol } = getSubdomainParts(request.url)
-    const verifiedFetchUrl = getVerifiedFetchUrl({ id, protocol, path })
-    log('verifiedFetch for ', verifiedFetchUrl)
+    log('verifiedFetch for ', event.request.url)
 
     const headers = request.headers
     headers.forEach((value, key) => {
       log.trace('fetchHandler: request headers: %s: %s', key, value)
     })
 
-    const response = await verifiedFetch(verifiedFetchUrl, {
+    const response = await verifiedFetch(event.request.url, {
```

Comment:
we no longer need a lot of the URL manipulating code (which was adding a trailing slash and not allowing verified-fetch unixfs dir redirect to work properly)